zio-cache/shared/src/main/scala/zio/cache/Cache.scala

```diff
@@ -59,6 +59,14 @@ sealed abstract class Cache[-Key, +Error, +Value] {
    * Returns the approximate number of values in the cache.
    */
   def size: UIO[Int]
+
+  /**
+   * Retrieves the value associated with the specified key if it exists.
+   * Otherwise computes the value with the lookup function, puts it in the
+   * cache, and returns it.
+   */
+  final def get(key: Key): IO[Error, Value] =
+    getAll(Chunk(key)).map(_.head)
```

Comment:
Since the behavior of `getAll` depends on the implementation of the `lookup` function, which is supplied by user, perhaps we should guarantee the correctness by using `headOption` in place of `head`. Obviously that would be a breaking change though.